subprojects/base-services/src/main/java/org/gradle/internal/classloader/ClasspathUtil.java

```diff
@@ -159,4 +160,13 @@ private static URI toURI(URL url) throws URISyntaxException {
             }
         }
     }
+
+    public static void getClasspath(ClassLoader classLoader, final Set<URL> classpath, ClassLoader stopAt) {
```

Comment:
This feels a little awkward (doesn't return a `ClassPath` like the other `getClasspath()` method, doesn't return anything even though it starts with `get`, `collectClasspathOf()` does essentially the same thing, etc).  At the least, I think we should just rename it to `collectClasspathUrlsOf()` or something like that.

Reply:
yep, I didn't like this name particularly either.